### ISSUE TITLE
Conditional directives around internal keyword

### ISSUE THREAD

**Issue by SteveGilham** [fsprojects/fantomas#628]
### Description

Failure as above with version 3.2.0-beta-002

### Repro code

Haven't tried to cut it down

[here it is](https://jindraivanek.gitlab.io/fantomas-ui/#?fantomas=preview&code=MQSwZgBASgqgcnAolAsAKAHYEMC2BTAZwAcsBjPCAQQBsAXAYQHsA3PAJwDoAhLAvdYHmp902fMTIUaDFuw5Q8pRmwAm7AXgwrw6dIyKaIAZQCeBWnhx6DGY2Ys4OTatUW0QjDAQ4BxTexBSa0NTc0tfakYAIyxqEAAvLHdPYNtQhw4ASQB5VLswxwANHGpdNAB6cogAFQALQgolLRBk7BcTCBAMCzY2iDBlCD8tLDZ8dFoTAwFwCAARRC4YHw1hfjQITc2unraNZrB0LeOTzYUiZVoAMWUcJIgAXiOIAB8IOCZWNkeIAAZnt7ZGyfdg-ACMAIgQM0ILYAHUWrVqmwyABrLoAcx+ACYypNpmhQJAFksVoShCINscduw9uSDs9TqcjLgiHEMFinlS3jRqD9-tzjJjXOC8VN1kT5otlqtKScab1YvttIcqUyTtUsJzIXAAK4ufmQ6ogfCiwX0WJ8h4QXGCriMWi1H4AZiNWCiIutABYypKkNVbQBtAA86XCUF13RNeCy3XY+iM7GYgUIHAACmxGBjMipKLRaGwQFFdRYABQAIkQAA9SNRdWorpmcEw1LCtRRamj+oNoRhYZ0CARdRQAGxggCs5YAlAA+AC6svWIbDjjmIC1GEY5kC3hbeEobTMIG81dr9bwjcYzcYrdkKIxeHnyp0aBDcEYTBwJELBE8T7Q+IUAqfRplgIDfFymwAN41NGEAAFydN0I5egA3BAFoGohOwQAAvr6sz+kGob2OGkbuPgsY9AmSYpt4GZZjmeYFkWJZ4BWp51g2TZ7m2D4QJ2pCot23y9v2x5DqOE7Tv+ghrOgy6kau64Ypu26kLuN77oeBDHhwnHnpe163l87ayZoKoKcG76ft+x5-guAHikhuyxDUKJCY8Or6qUgrGqajCQDsKGQphfKBS5kL2o6EARaB4Fuh6FARWupCtKMJjBuYhYcgANPMgTpWwmU7PlaaMDsABqx4tDOM7oFgWgQG+H5XnZv4YLJhGIAGVkrvI5HRlR8ZEImbDJuQ9GZtmub5oWxZlpWNZcRePFaXxHZdgMonAneA6SRAY6TrOjlyZSikFBwa4blu7gaU4WkHrER4nsthlrSZ7BmadFkvtScaKny5VVTVtBeWqEBQYyWw4CW7oikw5EIS5IUQ5syJogQyMADLHrQwYY0J9UQ-hEMAO6ItD5hJIEED4DgUSgvQbB4EkFCllOPwwYj3T8tj6GE6iWPWrj5gEx5qIzhzeFU7QNOkHTliM98mQYC0EC2FaECuGDtBflzEDA901W6bQTgs2z0sUzFPNg9atik+qTvO5setEBwgveJQKgqFAjX8b5LtB6cbvQ-TysQHw1BgBwRgWEQ0vWv1dQW9oHLDWwkRCXgKixqQLP4N0pYAGRRzHtucy8M6dKpyjrMc4egmXHsS5HV54I6mI-FnwnN57EClmAkYD5zAC01d9xLXs+23+CdxyU5h0rTdCDH1QOrEieR6vD1IwA1CjXrb9HLeY7v3RlDgN76kBAN9JQg54+Dmw6y5tJuSyYB4GuxBbhQVbPy2AWEw0NNilgAfBaumQf4XAIPDPAU4rrHlgexRe5NEQIQAPxQiiAAKzcDAv+KhTx4CIK0CA48R6X2viKYCblbagkghASoVQsq6hwHcYqxNNgsJqPUWKYAwB8DBmAagWoRIQCiCiDApAnRKHIgQZ4vDgzlCHBwjK3DtYdzfoDCAXBpGyKuGIzkfwqwAA5fiWKsQKZ4r86F8n0Y1WRABZXgwlrS-CrAAdiuL4vxVwyg8MqBANh6iuFKOCXUCgFEKD3DJrUQIci7ztggGwYejMMSNQiaw1R7DOEmE0a-WGcsko6L6NTNgtB-IUGtHMNm1SOAwFoKQd8ZNAnMOCaE-JmjeFRP6F0Y8CSOQQBiSMrAqJDARUdI0ZJ-E0kYGySE3JYSCm2O0cU+BZS3L4F4LqFm1Sfh1IsA0ppLTGBtLQIsrpGjFnY0YFgFQIz+GFDmIwUg7DNC0Cucs7pVzvy4A1rgPADxywkEdOWGcfTCjOOxv0kUtBGDa3uSoFR-ycA9M6QXUYQsZw5g8DgEZiKybKGEtbJ09BKDYmxL8bE6EyYUG3AaFQjBdSlO0L-PgTyGUFlZjgfKC0IBCHPFy1JN8OCLMyJATuWNjyCq+BALAEAiCZiSnyiA9L1UgANBgPAOc6ZwyiFqloHRGqPNoMSyOPLcAEHymSwVVY2SBBaIs7KvKIDstgW5WYprYoYGoB0WY0zUmszUN8QYZNCw9HdYwQgGAADkutxnJTJjqtgBAElEEWZQIgyrZCPLJlgANWNdQYn9RQ0eAl8xEAIPBSozL7o4ECJmX8YAzZKBwOUTQo9dQEHKMmIcsRzD1g8OUJQahR4AEddSxGNaOrAVKaWYOTHgMmDxmAEFHtSsEABOLNwhGD5WlaK2hWM1DKsUGzFQ3ysVsBxWssGyqQDMDZtAENLy3kfKLq6gFiE44WxwJzJhL9tH1p+MUagrz3mF1oBzaG9aOB3IeaWb9AG4NvPacotRvyqS8LhJG6JzzIOfq+ThzpPybmkdYWiwF+AQVKFMg+Ij0GIVQphdGqDnzCXqvw6i0YuAMVUb4wSsQwLQVJFqCx55bGwBauiUS-DhLeMonRVcm9OK8VXi48S29mqbaUupdiBVWN3SyBUWpgghTtGPufRYCAeGWh4HfRx3mpZ6NfUYx+6DyNwNMc+ZzZDVqCW-sC4B6Gbn7zf08582OWBWABf-Wg+9Wy+RXC6CoVWagqxGRgMQEAjzRHiJ7UQPLgDgMiPygrIDWxVa0GdNiFuJhQJpvrkyUsRW8v5X6j4SIMQ4iJFaBwPUDN2BxxMK4bwNW8APjYLlUBxwus9Znf1jwGAnD6loHsvAqsBixmfYWRqDB1ubbQdSSAUr6i2Fc68F4bwCsYhO1sCkFBR4QkuVSOxd83JiTvOB0rA9yyVGcTQvA5Z8rlk7Om0HEByz0DEYOQg5RYe8F7c4jutQbwo7RzeUHoDAz-cTFOzQ5Aja0F7QT4cMi8Ak6h78B7JwKyOJkbUEnvbGeyOp-lNntQjFak5nOMHzBSDTnaR91yfIPg-ZKH9islQr4qBvlD8scub45kV3PdHKgod44rHwCdRBug075wL0G8iDeJfe9o+xEBwNcBMDcMY9xtp3DtmHJIsiRLO907UaGbxziXHt87jg32vgIkdILTEPvX0XEqQHpIQfdrysocH9g4HI8AH0KHVwl18VPb2gmsKwxR-PxhYsMrwITynSqKq8wSWDU3pOuNVilyzaPIjZPfML+EyjITqMiZBbX22BBJMzIYxQFmQ46BYwRUhJQbBo9syU-xv5QmaOifC+2VLrhh8QHX-xGT8LEW6iICoF9vAFWWv-YvlTFuwZW5gMftmfvKkD1gbQBik0saIR81F3mlDIwtDpy2FLAuB2Hf0IE-2txKEQFcC80oVFnxkFmrj-zVloE5nOQWTRgHnrwgNSiKkymykxHylwJWwymDFKkNmr1oBNlqhnE5id3uF3wvHb0wNLBZVoCIBLE3woGCwSz+1flER7VqDjlGDtnmHqSGlOVaWhlfkYN815mtAUAeSc2Ix31mSYNcGkO0UzAdB+FkJ-zNjkNoGgMsE+XaW2DO1uHuAUNIX90sLNmz1BGmQwNOFfgqSqWjGYgglSUYAdF8A7k8NYkWjcOqWF0wKKVZiHH2Q8Lmh+G0LNj8FoACIWnYiVwiM2xCMALK1imoBUGEMqQOVqXEMoiaz4GrDIBg2COiILDB0YChwwB8kyK0TBkYByNR12SiNNEKOOSGhKLwDKLSlLB2UiLwGqU8JqLqIaLm0qNNGDArVLFmGmIoGDGyNyLlnyLgicMtREOqTm2DkFTWBWLyPcPwEQXXhgDVi+DgWoGqWlmCWcS6DmyGPSLglmIHlmCeI6IoGrhaJUDaOGIOU2I+JGOjF2L2P2M5R+L+OeJOI9kYHOKfXYCuJuM5juKwCrDmziNjn8LmkCJa1OB1zWOOJB1m0wK2EWNhL-UxArFqM6yUgiGiCWxpk8DWzoE2220YF21GHXG6HoCOxZinEaMxMTESJxOSNBIrCBJCJJKdklKGnXkpI5GpKhwWwZL6yZNW15NZJZnZM5P2x5L5IQUFJ8LNmFKSLYnFPLBUELC+EqkRJWxx1JM2HLGkFhHkEUGUFDWh1BIgAPn6mulrnUm8C4NtLTRWx1ISJDN0k8HZNLG9PmzpIUFETcBWw4Afj4AZn9T8KMKrEUBLExDTKVn9Q5gQzeXVMQWDLtM8HN2OFfkGPyhAHyhwDjXyhtQgGYEA0gOoFt1jxETsLMNUNH0MKxNcDSnfDUAIEGMAKrmMHLw4FIF4HxnA2MOg00U2GnIJw4DuCIAHiHlsCEEz32KzLNLLBACnHyiECnOrg3P3yjUHmHlLFRHyjTzHmrmwIem6DAi8AAGk8AOhURLyZyJ0OAHNvg7zLtHyFVBE3Ac4gd5dXAXzvSilgcAAJFoYWVQhRDgQMVERyJ2YJV5eNMGZlJ5WVLoX1CgVwLAbcuICZctb04JLAKCtKGC4HK6QgcgEYboCcwoOAIFLMnXcvfXWgacRorYYJBIkVdXDHQ9C7foOIbc6ZBKR0jpVQuMLiyRPAbaGZGRNmbAdwDkRC7RDAWwa0Rir+Zi341ixMEc2gMcwgQYuNUSzYYy709c2c+csWJcmAz5Vc04NyoCpQFwNwHc+8qSx5L-KA7y7oBC5Sk4RRWK44fyuclomykKy7ETfKO7GKsEhuLHXOaytwOyickTJyl2JKjyxcyKkw7oXysEpKrctKiAABShcBTK4xAUuMp2JKhFeAzq9Uac+AjgFmds0q+qqikARq+vUEYA2gNq3nTPPqsBQYt3J0egsGMlRarYX3GwmPOw+PGEO8UPJECWLuba1vHs-avsPaceTa44csdrTXW6ra6uMhI80U80hKpkac0sVLLQDLPALLJsHLYrfLdqp6t4DPShKatgM8pVVA0aq82cm86aoRMGOqOmFCtC98uWLoAgH8kwBGwC4C283cpqhaz6rIshH4AgJqdEimpojCgGH4VGpqza1+QXP7IOGrFCBrXop6sBV6hI489ids2ky6brVUhIdUwbdhZWUbcbYaabaUnKzYFU3rKWgbTUjbbUjAHbVWPbbkw7LUhBV4CeLQTayJC2MGUeVgNs0GRmhvUsRqe1SwMhFbBVH2PVMixqZ6XSAgTmM-eIeMNm7RevH4ZXVwVC0nLC6G3C4Odm0GdC5agBX4WFQXTmA+evWEuWagWDem160096ssZgfKftLG+UliRUrWtk3Wjk-Wrkg7au-k0qk4EA7oMAwcOGh20+ISf2wA1-DurGPQ5zEjaGYJFkW2uXcAEAPVJvPkafRVffPE2KEsDg64WTLEvAVEUsVOzrXVVEbIQsDELobgKbLoSuauEAWuFmaGNgtergrE2gbGTQDEGKVO6GWYdArlWwezCwZQrzYelQu+zg5g44AQ9NI49pC6DIf0tSO6TSNQJ6f1P22OI-c9QcVHQcdsCse4-OLcQKM2BQOId0I1SYB0lWihlWmHSlKxeCQhPgbIPBNwAgLgTSuuO5XSDkIwJQAwchyh-hoOAAKR7XcBk3nPIWtHLDjU1DYAfFoDjQHHdWQSIRkjjtF3fj5FeSMUEKr3MEHqr1AMzA-1ihTQdqxjWuDVbxXvYJLD+x7RH3c3UJqTmx1TJggC4L-V5VLBb0uHyi4NgpjF7D8dk0oFIA-zdIeV-rwGCdcCME7BZkG08GiYgC9F+G3RHGVsoa4KBFaG8HJ0+XXGoG4camrK2HsbGVke0Sq02Gd3d2AfWspkwLeHfB1QPNcYgFRyvmKk8dwGlkgWMECzmzeCMHbn6HJvVHaY8cC0HhibwACcusPuZlZgsFmdCfCaidmbidGBjBaeSdSfScyf4eybds8DyfLwp3cFiGKYwE5n6Z6asDzy2Ffnqa4M5s6EgByZTOPBGYClXtsc2LWLkbmyewHMcdebucGceayPAaEMJLefvxPwsCfwfVun0bbrfyMfAJMdsGwI9wYLUNeZedAa2GqSMBIAwFLCOWBMokkPOQ9kCCFgoTksEKOPpd7tKdfmsxfUQC8E21gpviwJZXUsQmIM8FIIILygKjShIOKjIO6DKkoOoNoDqk5gjq20eSq1mE3DBlcyFejqYA-NxvxoxrgvVc5icLmx7jMcaulhusdK1Z0N1cwoNZxu-N-JNZVxUCcs2LfO9hUEGOBxzCIMKhlZKnlYoJBlNilg6swI5as2tO5d5ZZhJ0FYUWRlFewFlfIJJyVdoIEhaBJxzD+wdZ1bfJdc-Lxvddr0La9e-stbeWElxbApHnGflEgG1dTa4uxoreNercoJzEaJ9b1enn9b7Z2CDYjeNlBnNmWdQRjaSytz9c1FKSdeFalbwKykroxGDelbFazfDZzdBhVf5tazBhFZDb3fwK3Z3Y3ezcVaPbzaqyKQNRFFrzDo8WxmhjNnxvisSsRqApAsaoJUoR5eGP5ZFFxYeYEZOFfh1SrFELfMDBwDjug6yPqBZh+DNiQ5Q9Q-Q5jB-dNpPfVA3MA+be3JA6Tap0oI1gBofSI9Q6ee0WYB+Dg6w6IBw4Y5VtfgeQ1YEnYBjEDHY-o847fb1ZCQrVE-3g9tznXhzrzs4-4ateY+bdtermYHPjBlePU9th9Ok40+E4U5yvU89lTJ9j9g5HYh457qFhbpyoFMAMk4vkefUd0T9aMGFDwCVbAVxfPd3czavZym3fXdDbldmsnaoIfcaLVaLbHe6CLaaAsHg7+1A75eB2tei6vQhlg9o90OHawoy5w5S+Teo9Y-zbf37cy80LBmY-tlo6wti9oBzBw9qaSTjCS42ohmaZ8gPPU7jlIXk62tnjRy7koWM9bl-HVwjze0lFgAQGQAXc+z5D9a89XYb1843YlaC4zdILvcjZoKi8DceQa-i88ES5dwhha9UrO690j2XfhQPKXc2VxZI0SogEhurjc48688g8O7K5rau9o86rBE-fpBVCAA&config=N4IgkgdgJgphAuBlADgQwMYwHIFcC2IAXAEwA0IACqgOYwDqAllPABZECcADAL5A)

**Comment by nojaf** [comment]
Main problem here is the trivia printing for 
```fsharp
type
#if DEBUG
#else
     internal
#endif
              ReportFormat =
  | NCover = 0
  | OpenCover = 1
  | OpenCoverWithTracking = 2
```

**Comment by nojaf** [comment]
Fix available in [v5.2.0-beta-001](https://www.nuget.org/packages/fantomas/5.2.0-beta-001)
